spline-shadow/client-ui/src/app/components/lineage/lineage-graph/lineage-graph.component.ts

```diff
@@ -48,11 +53,28 @@ export class LineageGraphComponent implements OnInit {
         })
         that.cytograph.cy.add(response)
 
-        that.cytograph.cy.nodes().on('click', function (event) {
-          let clickedNode = event.target
-          that.lineageGraphService.getDetailsInfo(clickedNode.id())
+        that.cytograph.cy.on('click', function (event) {
+          const clikedTarget = event.target
+          let nodeId = null
+          if (clikedTarget != that.cytograph.cy && clikedTarget.isNode()) {
+            nodeId = clikedTarget.id()
```

Comment:
Please prefer more functional without reassignment. E.g. `const nodeId = `clikedTarget != that.cytograph.cy && clikedTarget.isNode() ? clikedTarget.id() : null` if you like.